Doppler.HtmlEditorApi/Repositories.DopplerDb/DopplerCampaignContentRepository.cs

```diff
@@ -71,11 +72,19 @@ public async Task<CampaignState> GetCampaignState(string accountName, int campai
             : campaignStateData.Status == DopplerCampaignStatusInWinnerInABSelectionProcess ? CampaignStatus.InWinnerInABSelectionProcess
             : CampaignStatus.Other;
 
-        return new CampaignState(
-                campaignStateData.OwnCampaignExists,
+        if (campaignStateData.TestType == 0)
```

Comment:
Why did you create a constant for `1` (`DopplerCampaignTestTypeSubject`) but not for `0`?

Reply:
Because I'm assuming that will be only two values. So if it's not 1, will be byContent

Reply:
Solved